weather-wise/src/main/java/app/weatherwise/APIService.java

```diff
@@ -18,7 +17,7 @@ public class APIService {
 
     private static APIService instance;
 
-    public APIService() {
+    private APIService() {
     }
```

Comment:
Fixed singleton issue by correctly making the constructor private since getInstance() is already public.